SyntheticDataset/SyntheticDataset/image_operations/ImageBounder.py

```diff
@@ -5,6 +5,26 @@ class ImageBounder(object):
 
     @staticmethod
     def getBoundsOfColor(img, image, color):
+
+        dim = img.size
+
+        list_of_x = []
+        list_of_y = []
+
+        for x in range(0, dim[0]):
+            for y in range(0, dim[1]):
+                if  image[x,y] == color:
+                    list_of_x.append(x)
+                    list_of_y.append(y)
+
+        leftX = min(list_of_x)
+        rightX = max(list_of_x)
+        upY = min(list_of_y)
+        lowY = max(list_of_y)
+
+        return Rectangle(leftX-1, upY-1, rightX+1, lowY+1)
```

Comment:
This is not correct. If you look at the rectangle documentation, you need to pass the top left point, then the width and height of the rectangle (not the bottom right point).